### PR TITLE
User Cluster MLA e2e test

### LINKED ISSUE #7890: UserCluster MLA E2E tests: Deploy, Grafana Org and Datasource
Test that MLA deployed and check that corresponding Grafana Org and Datasource created

### PR DESCRIPTION
**Pull request by aborilov**
**What this PR does / why we need it**:
In this PR we added UserCluster MLA e2e tests:
* deploy UserCluster MLA stack
* check that Grafana Org added
* check that Grafana Datasource added

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7890 

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ presubmits:
   #########################################################
   # unit tests
   #########################################################
-
+
```
Comment (xrstf) [inline .prow.yaml]: Please remove all of these excessive whitespaces.

File: pkg/test/e2e/mla/mla_test.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
+//go:build e2e
```
Comment (zreigz) [inline pkg/test/e2e/mla/mla_test.go]: you can change tag for `mla`

File: hack/ci/run-mla-e2e-tests.sh [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+spec:
+  email: roxy2@loodse.com
+  id: 1413636a43ddc27da27e47614faedff24b4ab19c9d9f2b45dd1b89d9_KUBE
+  name: roxy2
+  admin: true
+EOF
+retry 2 kubectl apply -f user.yaml
+
+echodate "Running mla tests..."
+go test -timeout 30m -tags e2e -v ./pkg/test/e2e/mla -kubeconfig "$KUBECONFIG"
```
Comment (zreigz) [inline hack/ci/run-mla-e2e-tests.sh]: you can change the tag for `mla`

File: pkg/test/e2e/mla/mla_test.go (hunk 2 above)
Comment (zreigz) [reply pkg/test/e2e/mla/mla_test.go]: you still have e2e instead of mla

File: .prow.yaml (hunk 1 above)
Comment (aborilov) [reply .prow.yaml]: done

File: hack/ci/run-mla-e2e-tests.sh [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+retry 2 kubectl apply -f preset-digitalocean.yaml
+
+echodate "Creating roxy2 user..."
+cat << EOF > user.yaml
+apiVersion: kubermatic.k8s.io/v1
+kind: User
+metadata:
+  name: c41724e256445bf133d6af1168c2d96a7533cd437618fdbe6dc2ef1fee97acd3
+spec:
+  email: roxy2@loodse.com
```
Comment (xrstf) [inline hack/ci/run-mla-e2e-tests.sh]: Please use @kubermatic.com (cf. #8019)

File: hack/ci/setup-kubermatic-mla-in-kind.sh [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+# NB: The CE requires Seeds to be named this way
+export SEED_NAME=kubermatic
+
+# This defines the Kubermatic API endpoint the e2e tests will communicate with.
+# The api service is kubectl-proxied later on.
+export KUBERMATIC_API_ENDPOINT="http://localhost:8080"
+
+# Tell the conformance tester what dummy account we configure for the e2e tests.
+export KUBERMATIC_DEX_VALUES_FILE=$(realpath hack/ci/testdata/oauth_values.yaml)
+export KUBERMATIC_OIDC_LOGIN="roxy@loodse.com"
```
Comment (xrstf) [inline hack/ci/setup-kubermatic-mla-in-kind.sh]: @kubermatic.com pls

File: hack/ci/run-mla-e2e-tests.sh (hunk 4 above)
Comment (aborilov) [reply hack/ci/run-mla-e2e-tests.sh]: done

File: hack/ci/setup-kubermatic-mla-in-kind.sh (hunk 5 above)
Comment (aborilov) [reply hack/ci/setup-kubermatic-mla-in-kind.sh]: Done

**Pull request merged**